docs/modules/integrations/pages/webhooks/create-and-manage-webhooks.adoc

```diff
@@ -0,0 +1,128 @@
+= Create and Manage Webhook
+:navtitle: Create and Manage Webhooks
+
+Learn how to use Kobiton webhook to send a notification via HTTP to another application when an event occurs.
+
+You can use Kobiton webhooks to notify your app or an internal system when certain events occur in Kobiton. For example, you might use webhooks to alert your remote application when a device has gone offline or becomes available instead of periodically polling Kobiton via REST APIs to determine whether changes have occurred.
+
+Webhooks can be set up by org admins or any role with the *org_setting.modify* permission enabled.
+
+== Supported webhook events
+
+A webhook is triggered to send an HTTP callback when a particular event occurs.
+
+Kobiton currently supports creating device status webhooks for the following events:
+
+. *Online*: from any status, except utilizing, to *online*
+. *Offline*: from any status to offline
+. *Utilizing*: from any status to utilizing
+. *Unplugged*: from any status to unplugged
+
+[NOTE]
+===============================
+Device status webhooks are only available for *Private* and *Local* devices. They cannot be used to report status for *Public* devices.

```

Comment:
Question for @tungmhoang - are webhooks available for Standalone? Do they require an internet connection? Or can they be on a network?

Reply:
Webhooks are available for both Standalone and Cloud, as long as the customer machine can connect to the Dell server to receive webhooks.

Reply:
@erincbailey do you think we need to explicitly mention the point above?